<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>613</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>782</width>
      <height>404</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>780</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>300</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>182</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2" colspan="2">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>240</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="3" rowspan="4">
         <widget class="QPushButton" name="Bt_openDialog">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-image: url(:/prefix1/image/set.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2" rowspan="3">
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>208</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>208</width>
            <height>332</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>507</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>780</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>50</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox_qrcodeType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox#comboBox_qrcodeType {
    min-height: 32px;
    min-width: 96px;
    border: none;
    font: 14px &quot;Arial&quot;;
    background: rgb(255, 255, 255); /* 橙色背景 */
    color: rgb(0, 0, 0); /* 文本颜色为白色 */
}

QComboBox#comboBox_qrcodeType QAbstractItemView {
    min-height: 32px;
    min-width: 96px;
    border: none;
    font: 14px &quot;Arial&quot;;
    color: rgb(0, 0, 0);
}

QComboBox#comboBox_qrcodeType QAbstractItemView::item {
    min-height: 32px; /* 调整下拉选项的高度为32像素 */
    min-width: 96px; /* 调整下拉选项的宽度为128像素 */
    font: 14px &quot;Arial&quot;;
}

QComboBox#comboBox_qrcodeType QAbstractItemView::item:hover {
    background-color:rgb(0, 0, 0);
	color: rgb(255, 255, 255); /* 文本颜色为白色 */
}

</string>
          </property>
          <item>
           <property name="text">
            <string>普通二维码</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>交流桩二维码</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>88</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Bt_generate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">/* 设置 Bt_generate 按钮的样式 */
QPushButton#Bt_generate {
    min-width: 86px;
    max-width: 86px;
    min-height: 22px;
    max-height: 22px;
    border: 1px solid #0078D7;
    background: rgb(255, 255, 255); /* 橙色背景 */
    color: rgb(0, 0, 0); /* 文本颜色为白色 */
    border-radius: 5px;
    padding: 4px;
}


QPushButton#Bt_generate:hover {
    background-color: #efe6e0;
    border: 1px solid #efe6e0;
}

QPushButton#Bt_generate:pressed {
    background-color: #8f8f8f;
    border: 1px solid #8f8f8f;
}</string>
          </property>
          <property name="text">
           <string>生成二维码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>268</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="Bt_save">
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>32</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#Bt_save {
    min-width: 86px;
    max-width: 86px;
    min-height: 22px;
    max-height: 22px;
    border: 1px solid #0078D7;
    background: rgb(255, 255, 255); /* 橙色背景 */
    color: rgb(0, 0, 0); /* 文本颜色为白色 */
    border-radius: 5px;
    padding: 4px;
}

QPushButton#Bt_save:hover {
    background-color: #efe6e0;
    border: 1px solid #efe6e0;
}

QPushButton#Bt_save:pressed {
    background-color: #8f8f8f;
    border: 1px solid #8f8f8f;
}
</string>
          </property>
          <property name="text">
           <string>下载图片</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(87, 182, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
